scripts/solana.patch

```diff
@@ -25,15 +25,7 @@ index 541629ad49..8c2ec94041 100644
    -Werror \
    -O2 \
    -fno-builtin \
-@@ -68,6 +77,7 @@ BPF_CXX_FLAGS := \
```

Comment:
why did this change?

Reply:
We're actually only changing `-z defs`. Thus the `BPF_LLD_FLAGS` is no longer modified, so it doesn't appear in the patch file.

Reply:
the point of this patch was to ensure that the program can be loaded by the runtime

I believe we already had an issue where pyth was down due to having an unresolved symbol